pkg/core/plugins/registry.go

```diff
@@ -30,11 +30,13 @@ const (
 
 type Registry interface {
 	Bootstrap(PluginName) (BootstrapPlugin, error)
+	BootstrapPlugins() map[PluginName]BootstrapPlugin
```

Comment:
Do we still need `Bootstrap(PluginName)` and `Runtime(PluginName)`?

Reply:
Left it out of the convention, I can remove it